Doc/library/datetime.rst

```diff
@@ -1804,16 +1806,20 @@ When DST starts (the "start" line), the local wall clock leaps from 1:59 to
 ``astimezone(Eastern)`` won't deliver a result with ``hour == 2`` on the day DST
 begins.  For example, at the Spring forward transition of 2016, we get
 
-    >>> u0 = datetime(2016, 3, 13, 5, tzinfo=timezone.utc)
-    >>> for i in range(4):
-    ...     u = u0 + i*HOUR
-    ...     t = u.astimezone(Eastern)
-    ...     print(u.time(), 'UTC =', t.time(), t.tzname())
-    ...
-    05:00:00 UTC = 00:00:00 EST
-    06:00:00 UTC = 01:00:00 EST
-    07:00:00 UTC = 03:00:00 EDT
-    08:00:00 UTC = 04:00:00 EDT
+.. doctest::
+
+   >>> from datetime import datetime, timezone
+   >>> from tzinfo_examples import HOUR, Eastern
```

Comment:
I have just added the two lines above, and deindented all the example of 1 whitespace, to fit the doctest directive indentation.